.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we need to be committing vscode settings?

Reply:
I will remove it - it facilitates debugging.